src/evidence.cpp

```diff
@@ -757,29 +758,57 @@ void Courtroom::on_evidence_load_clicked()
             // mode for now
 
   QString p_path = QFileDialog::getOpenFileName(
-      this, tr("Open Inventory"), "base/inventories/", tr("Ini Files (*.ini)"));
+      this, tr("Open Inventory"), "inventories/", tr("Ini Files (*.ini)"));
   if (p_path.isEmpty())
     return;
 
   evidence_close();
   ui_evidence_name->setText("");
+  evidence_load(p_path);
+  set_evidence_page();
+}
 
-  QSettings inventory(p_path, QSettings::IniFormat);
+void Courtroom::evidence_load(QString filename)
+{
+  if (!file_exists(filename)) {
+    qWarning() << "Trying to load a non-existant evidence save file:" << filename;
+    return;
+  }
+  QSettings inventory(filename, QSettings::IniFormat);
   local_evidence_list.clear();
   foreach (QString evi, inventory.childGroups()) {
     if (evi == "General")
       continue;
 
     evi_type f_evi;
-    f_evi.name = inventory.value(evi + "/name", tr("UNKNOWN")).value<QString>();
+    f_evi.name = inventory.value(evi + "/name", "<name>").value<QString>();
     f_evi.description =
-        inventory.value(evi + "/description", tr("UNKNOWN")).value<QString>();
+        inventory.value(evi + "/description", "<description>").value<QString>();
     f_evi.image =
-        inventory.value(evi + "/image", "UNKNOWN.png").value<QString>();
+        inventory.value(evi + "/image", "empty.png").value<QString>();
     local_evidence_list.append(f_evi);
   }
   private_evidence_list = local_evidence_list;
-  set_evidence_page();
+}
+
+void Courtroom::evidence_save(QString filename)
+{
+  // "Inventories" dir keeps our private evidence data
+  if (!dir_exists("inventories")) {
+    // Create one if it doesn't yet exist
+    QDir("inventories").mkdir("inventories");
+  }
+
+  QSettings inventory(filename, QSettings::IniFormat);
```

Comment:
```suggestion
  QSettings inventory(filename, QSettings::IniFormat);
  inventory.setIniCodec("UTF-8");
```

Reply:
It does. This is why the fav list was broken when special characters was used.